docs/content/app-protect/configuration.md

```diff
@@ -19,7 +20,7 @@ The NGINX Ingress Controller has a set of global configuration parameters that a
 ## Enabling App Protect
 
 You can enable and configure NGINX App Protect on the Custom Resources (VirtualServer, VirtualServerRoute) or on the Ingress-resource basis. Refer to [Custom Resource](https://docs.nginx.com/nginx-ingress-controller/configuration/policy-resource/#waf) examples for more details.
-To do so, you can apply the [App Protect annotations](/nginx-ingress-controller/configuration/ingress-resources/advanced-configuration-with-annotations/#app-protect) to each desired resource.
+To configure NGINX App Protect on an Ingress resource, you would apply the [App Protect annotations](/nginx-ingress-controller/configuration/ingress-resources/advanced-configuration-with-annotations/#app-protect) to each desired resource.
 
```

Comment:
I am still missing / not finding a configuration example for a VirtualServer resource.
I won't let this go with out that.  Sorry.

All the example here and below are focused on the use of the Ingress Resource and outside of a pointer to the examples on line 22, the customer has the perception that only the Ingress resource is supported.
That one line call-out is not good enough.

Each place that the description outlines using the Ingress Resource there needs to be a corresponding description for the VirtualServer Resource.

Reply:
I've added the reference to VirtualServer Resource here (line 12) according to your last comment.

Do you want me to add the reference link here as well?

Reply:
Sorry, I was not clear. I am not referring to the samples link.

I am referring to the walkthrough itself.
The walkthrough is exclusive to using the Ingress resource.  Which leads the customer to believe this is the only method and a simple link that can be easily overlooked does not repair that perception.

There needs to be a walkthrough for both.

We place a high value emphasis on the CRDs, and thus want to highlight them first.

Reply:
Hi @liorm39 , could you please help here with this information?

Reply:
@brianehlert, I've added the Virtual Server example and Configuration details. Please review.